lib_dop/r_dop_import_lib.py

```diff
@@ -373,6 +373,16 @@ def import_and_reproject(
         tmp_loc (str): Name of temporary location
         tmp_gisrc (str): Path to GISRC file
     """
+
+    # since 8.4.0 use project variable instead of location
```

Comment:
Do we really need this conditional part? I strongly suggest to not use any GRASS GIS version < 8.4.0.